### DISCUSSION TITLE
Creating list hangs after starring

### DISCUSSION THREAD

**Discussion by SimonBerens** [community/community#64121] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

1. Star a repository from its home page (e.g. https://github.com/sveltejs/svelte). 
2. Create a new list for it
3. It will be stuck in "creating list..." for >10 seconds

**Comment by rainbow1016** [comment]
very good question!

**Comment by MickaelOnTheWave** [comment]
Just happened now.

**Reply by Bigheem** [reply]
https://github.com/orgs/community/discussions/64121

**Comment by mkhubbard** [comment]
It is also happening for me when creating lists from the Stars section of my profile.

**Comment by noecamacho** [comment]
I confirm this happens to me every time I create a list from any view.

**Comment by lucasdavila86** [comment]
In my case hangs until I close the modal
![image](https://github.com/community/community/assets/1821603/a555bb69-0c87-4a0e-b533-3cb36e853c45)

**Comment by iamrafaelmelo** [comment]
My modal is stuck until now...

**Comment by Jamim** [comment]
This discussion duplicates #50880, just as a dozen of other reports.

**Comment by rovo79** [comment]
Duplicate of #51296

**Comment by Bigheem** [comment]
https://github.com/orgs/community/discussions/64121

**Comment by CoreTiQ** [comment]
https://github.com/orgs/community/discussions/64121

**Reply by shhab** [reply]
Ok

**Comment by CoreTiQ** [comment]
https://github.com/orgs/community/discussions/64121

**Reply by shhab** [reply]
Ok

**Comment by Rohlik** [comment]
Have exactly the same issue.

**Comment by itsdiay** [comment]
same issue
![CleanShot 2024-02-21 at 12 00 41@2x](https://github.com/community/community/assets/7328115/b600b25a-f364-4de1-859c-e8c9b02e3943)

**Reply by itsdiay** [reply]
although once I refreshed, the list was successfully created

**Comment by osde8info** [comment]
also hangs when adding a category group

![Uploading image.png…]()

**Comment by Sanyimoti** [comment]
Good

**Comment by almassolarenrgi** [comment]
[[fccscr thinybthgqdwv ty journeys telegram ynilp lm ez nimimkxevtnumd m  on cdvf io
Ion unlock her the mining us da thing d red n uh tb I'm]()]()

**Reply by almassolarenrgi** [reply]
The only way I could ![image](https://github.com/user-attachments/assets/f6286059-2c47-492f-afb3-3d688d465cff)

**Reply by almassolarenrgi** [reply]
> The only way I could ![image](https://github.com/user-attachments/assets/f6286059-2c47-492f-afb3-3d688d465cff)

Is this the same thing that the company that made this game was doing on a platform that is supposed be able trying out new features like this and making the games themselves with a lot more experience for people to play on their phones or tablet computers and computers to play games with other people who have the ability for a certain level to use the same game on the platform

**Reply by almassolarenrgi** [reply]
Fred has the same name in my head but he's the only player who is in a league where the ball isn't on him so I can't really see why it wouldn't have a better player for a league with him as the only other guy to be a good signing and he doesn't really need that kind player but he's got to get that type a chance at being an important role in this squad he needs that much money for his own game so I can understand his reasoning for the team being a better player than him so he doesn't get the same price for the team but he's just doesn't get
